hyperspy/tests/signals/test_eels.py

```diff
@@ -268,7 +267,7 @@ def test_spikes_diagnosis(self):
         np.testing.assert_allclose(hist_data.data, expected_data)
 
         hist_data2 = self.signal._spikes_diagnosis(bins=25)
-        expected_data2 = np.array([285, 11, 13, 0, 0, 1, 12,  0])
+        expected_data2 = np.array([286, 10, 13, 0, 0, 1, 12,  0])
```

Comment:
Will this test fail if you have an older version of numpy installed which uses the older random state model? If that is the case it might be worthwhile to add a comment. 

Why are the first two values shifted by one if the seed is the same as before?

Reply:
This isn't possible because this is using the new numpy generators. It is still possible for the user to use legacy generators by specifying the seed and this would generate different results, but the default is using new generators.
In case of older version of dask, some tests are skipped for keep it simple.

Reply:
Sounds good!